opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/ListTaskInteractor.java

```diff
@@ -71,8 +74,8 @@ public void run() {
                 CardDetails cardDetails = null;
                 try {
                     if ((PAOT.equals(interventionType)) && isForForm && cursor.moveToFirst()) {
-                        cardDetails = new MosquitoHarvestCardDetails(cursor.getString(cursor.getColumnIndex(DatabaseKeys.PAOT_STATUS)),
-                                cursor.getString(cursor.getColumnIndex(DatabaseKeys.LAST_UPDATED_DATE)), null, PAOT);
+                        cardDetails = new MosquitoHarvestCardDetails(cursor.getString(cursor.getColumnIndex(PAOT_STATUS)),
```

Comment:
You either remove this block on remove `createPaotCardDetails`, both are doing the same thing